jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-1a.yaml

```diff
@@ -202,8 +210,16 @@
           # copy scripts/ dir onto the admin node in /root/scripts
           scp -r ~/github.com/$repo_owner/automation/scripts mkcloud.config root@$admin:
 
+          # Use custom scenario dir if specified, otherwise default to the one
+          # based on $cloud_version and $ssl_type.
+
+          if [ -z "$scenario_dir" ]; then
+            scenario_dir=scripts/scenarios/cloud$cloud_version/qa/$ssl_type/$scenario_file
```

Comment:
Remove $scenario_file as otherwise it is duplicated below. Same with all files changed in this commit.